code/code/sys/socket.cc

```diff
@@ -1810,7 +1830,7 @@ int TMainSocket::gameLoop()
   scheduler.add(new procFactoryProduction(Pulse::MUDDAY));
 
   // pulse realhour
-//  scheduler.add(new procTweakLoadRate(Pulse::REALHOUR)); // desired load rate achieved
+  scheduler.add(new procTweakLoadRate(Pulse::ONE_SECOND));
```

Comment:
Uhh, do we need to run it _every_ second? I do realize the checks in it are cheap, but still.

Reply:
We can probably change this to something else. Optimally, I would like to be able to remove this proc from the scheduler when not in use. I'd need to add a function to search for it and do that. I haven't looked very hard in how to access the scheduler instance from other locations besides the main game loop, but once we do that, we can just add and remove as we like.

As for it being cheap, if the change rate is set to zero, the first thing it does is returns, so it isn't much of a hit when not in use.